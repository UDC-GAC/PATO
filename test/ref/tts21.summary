# Duplex-ID	TTSs (abs)	TTSs (rel)
ENSLCOT00000000041.1	101	0.000829
ENSLCOT00000000044.1	101	0.000859
ENSLCOT00000000056.1	101	0.00089
ENSLCOT00000000007.1	487	0.0111
ENSLCOT00000000004.1	7	0.000617
ENSLCOT00000000005.1	1	0.00019
ENSLCOT00000000006.1	61	0.00246
ENSLCOT00000000009.1	18	0.000682
ENSLCOT00000000445.1	149	0.00303
ENSLCOT00000000484.1	149	0.0031
ENSLCOT00000000489.1	155	0.00323
ENSLCOT00000000017.1	37	0.00247
ENSLCOT00000000881.1	114	0.00175
ENSLCOT00000000921.1	113	0.00211
ENSLCOT00000000946.1	89	0.00219
ENSLCOT00000000013.1	30	0.00203
ENSLCOT00000000012.1	328	0.0069
ENSLCOT00000000078.1	27	0.000422
ENSLCOT00000000070.1	7	0.000346
ENSLCOT00000000130.1	4	0.000218
ENSLCOT00000000131.1	22	0.001
ENSLCOT00000000069.1	31	0.00129
ENSLCOT00000000025.1	1	0.000138
ENSLCOT00000000043.1	159	0.0203
ENSLCOT00000000016.1	102	0.0032
ENSLCOT00000000323.1	578	0.0115
ENSLCOT00000000346.1	467	0.0103
ENSLCOT00000000018.1	14	0.000615
ENSLCOT00000000035.1	36	0.00129
ENSLCOT00000000014.1	471	0.00616
ENSLCOT00000000034.1	19	0.00141
ENSLCOT00000000251.1	456	0.00303
ENSLCOT00000000255.1	438	0.00301
ENSLCOT00000000281.1	1	7.98e-05
ENSLCOT00000000030.1	244	0.0025
ENSLCOT00000000015.1	14	0.00194
ENSLCOT00000000024.1	102	0.00432
ENSLCOT00000000032.1	102	0.0047
ENSLCOT00000000019.1	4	0.000125
ENSLCOT00000000022.1	156	0.00965
ENSLCOT00000000039.1	42	0.00175
ENSLCOT00000000048.1	1	0.000172
ENSLCOT00000000402.1	54	0.00344
ENSLCOT00000000020.1	17	0.00111
ENSLCOT00000000353.1	11	0.000621
ENSLCOT00000000026.1	169	0.00171
ENSLCOT00000000023.1	33	0.00107
ENSLCOT00000000027.1	167	0.00773
ENSLCOT00000000028.1	7	0.000307
ENSLCOT00000000029.1	15	0.000561
ENSLCOT00000000038.1	7	0.000283
ENSLCOT00000000033.1	54	0.00327
ENSLCOT00000000045.1	21	0.00182
ENSLCOT00000000417.1	12	0.000602
ENSLCOT00000000452.1	12	0.000562
ENSLCOT00000000491.1	12	0.000592
ENSLCOT00000000080.1	118	0.00337
ENSLCOT00000000037.1	132	0.00668
ENSLCOT00000000036.1	24	0.00119
ENSLCOT00000000042.1	28	0.000523
ENSLCOT00000000093.1	156	0.00316
ENSLCOT00000000103.1	42	0.00162
ENSLCOT00000000169.1	6	0.000351
ENSLCOT00000000249.1	12	0.000394
ENSLCOT00000000259.1	12	0.000361
ENSLCOT00000000260.1	21	0.000574
