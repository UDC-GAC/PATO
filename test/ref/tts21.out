# Duplex-ID	Start	End	Score	Strand	Error-rate	Errors	Guanine-rate	Duplicates	TTS	Duplicate locations
ENSLCOT00000000041.1	7990	8010	19	+	0.05	d3	0.45	2	GGACAGAAGGAGAAAAGAGG	-
ENSLCOT00000000044.1	7711	7731	19	+	0.05	d3	0.45	2	GGACAGAAGGAGAAAAGAGG	-
ENSLCOT00000000056.1	7435	7455	19	+	0.05	d3	0.45	2	GGACAGAAGGAGAAAAGAGG	-
ENSLCOT00000000007.1	924	944	19	+	0.05	d2	0.6	0	AGCGAGGAGGAGAGGAAGGG	-
ENSLCOT00000000007.1	986	1002	16	+	0		0.56	0	GAAGGAGGAAGAGGAG	-
ENSLCOT00000000007.1	1173	1196	22	+	0.043	d18	0.57	0	GAGGAGGAAGAGGAGGAGCAGGA	-
ENSLCOT00000000007.1	1866	1887	20	+	0.048	d8	0.62	0	GGGGAGGACGAAGAGGAGGGA	-
ENSLCOT00000000007.1	463	491	24	-	0.14	d2d26	0.57	0	ACAGAGGAGGAAGAGGAGGAGGGGGCAG	-
ENSLCOT00000000445.1	2732	2757	20	+	0.2	d2d21	0.48	2	GGCGAAGAAAGAGAGGGAAAGCAGG	-
ENSLCOT00000000484.1	2654	2679	20	+	0.2	d2d21	0.48	2	GGCGAAGAAAGAGAGGGAAAGCAGG	-
ENSLCOT00000000489.1	2651	2676	20	+	0.2	d2d21	0.48	2	GGCGAAGAAAGAGAGGGAAAGCAGG	-
ENSLCOT00000000017.1	611	632	20	+	0.048	d10	0.67	0	GGGGGGGAAGTGGAAGGAGAG	-
ENSLCOT00000000881.1	751	771	19	-	0.05	d8	0.5	2	AGAGGGAAGGGCAAAGAGAG	-
ENSLCOT00000000921.1	568	588	19	-	0.05	d8	0.5	2	AGAGGGAAGGGCAAAGAGAG	-
ENSLCOT00000000946.1	568	588	19	-	0.05	d8	0.5	2	AGAGGGAAGGGCAAAGAGAG	-
ENSLCOT00000000013.1	954	972	18	-	0		0.28	0	GAAAGAAAAAAGGAGAAA	-
ENSLCOT00000000012.1	3103	3124	20	-	0.048	d9	0.62	0	GGGGGAGAGAGTGAAGAGGGA	-
ENSLCOT00000000131.1	11	29	18	-	0		0.5	0	GAGAAGGAGAGGGAAAAG	-
ENSLCOT00000000069.1	1495	1513	18	+	0		0.56	0	GGGAGGAAGAGGAAGAGA	-
ENSLCOT00000000043.1	393	418	24	+	0.04	d6	0.44	0	GGGAAATGGGGAAAAGAAGAGAAGA	-
ENSLCOT00000000043.1	483	508	24	+	0.04	d2	0.48	0	AGTAGAGAAAAGGAGGGGAGAAAGG	-
ENSLCOT00000000323.1	2151	2172	20	+	0.048	d15	0.71	1	GGGGAGGAGGAAGGGCGGAGG	-
ENSLCOT00000000346.1	1866	1887	20	+	0.048	d15	0.71	1	GGGGAGGAGGAAGGGCGGAGG	-
ENSLCOT00000000014.1	231	257	25	+	0.038	d21	0.38	0	AAAGAAAAGGGGAAGAGGAGATAAAG	-
ENSLCOT00000000014.1	270	295	24	+	0.04	d9	0.4	0	AAAGAAGAACGAGAAGGAGAGGGAA	-
ENSLCOT00000000014.1	1424	1446	21	+	0.045	d14	0.55	0	GAGAGGGAAAGAAATGGGGAGG	-
ENSLCOT00000000014.1	3593	3614	20	+	0.048	d13	0.57	0	GAGGGAGGGAAAATGGGGGAA	-
ENSLCOT00000000014.1	773	797	23	-	0.042	d21	0.5	0	AGCAGGAGAGGAGGAGGAAGAAGA	-
ENSLCOT00000000251.1	1298	1319	20	+	0.048	d8	0.38	1	GAAGGAAGCAGAGAAAGGAAA	-
ENSLCOT00000000251.1	4292	4308	16	+	0		0.44	1	GGAGAAAAGAGAAGGA	-
ENSLCOT00000000251.1	5031	5053	21	+	0.045	d7	0.27	1	GAGGAAGCAAAGAGAAAAAAAA	-
ENSLCOT00000000251.1	6533	6558	24	+	0.04	d20	0.48	1	GAGAGGAGAAGAGGAAGAAGTGGAA	-
ENSLCOT00000000251.1	7805	7826	20	+	0.048	d4	0.24	1	AGAACAAAGAGAAGAAGAAAA	-
ENSLCOT00000000251.1	1989	2010	20	-	0.048	d18	0.43	1	AGTGGAAAAGGAGGGAAAAGA	-
ENSLCOT00000000255.1	1130	1151	20	+	0.048	d8	0.38	1	GAAGGAAGCAGAGAAAGGAAA	-
ENSLCOT00000000255.1	4124	4140	16	+	0		0.44	1	GGAGAAAAGAGAAGGA	-
ENSLCOT00000000255.1	4863	4885	21	+	0.045	d7	0.27	1	GAGGAAGCAAAGAGAAAAAAAA	-
ENSLCOT00000000255.1	6365	6390	24	+	0.04	d20	0.48	1	GAGAGGAGAAGAGGAAGAAGTGGAA	-
ENSLCOT00000000255.1	7637	7658	20	+	0.048	d4	0.24	1	AGAACAAAGAGAAGAAGAAAA	-
ENSLCOT00000000255.1	1821	1842	20	-	0.048	d18	0.43	1	AGTGGAAAAGGAGGGAAAAGA	-
ENSLCOT00000000030.1	3971	3991	19	-	0.05	d14	0.45	0	GGGAGTAAAGGAGAAGAAAG	-
ENSLCOT00000000030.1	4873	4899	26	-	0		0.31	0	AAAAAGGAGAAGGGAAAAAAAAAGAG	-
ENSLCOT00000000024.1	831	849	18	+	0		0.22	1	AGAAGAAGAAAGAAAAAA	-
ENSLCOT00000000024.1	1071	1091	19	+	0.05	d5	0.3	1	GAAAATGAAGAAAGAAAAGG	-
ENSLCOT00000000024.1	1377	1400	22	+	0.043	d3	0.3	1	AGATGAAGAAGAAAAAGGAAAGA	-
ENSLCOT00000000032.1	831	849	18	+	0		0.22	1	AGAAGAAGAAAGAAAAAA	-
ENSLCOT00000000032.1	945	965	19	+	0.05	d5	0.3	1	GAAAATGAAGAAAGAAAAGG	-
ENSLCOT00000000032.1	1251	1274	22	+	0.043	d3	0.3	1	AGATGAAGAAGAAAAAGGAAAGA	-
ENSLCOT00000000022.1	5	35	26	+	0.13	d8d27	0.47	0	AAGAAGAGTGAAAAGGGGAGAGGAGGGCAA	-
ENSLCOT00000000039.1	56	77	20	-	0.048	d15	0.33	0	AAGAGCAGAAGAAAGAGAAGA	-
ENSLCOT00000000026.1	6307	6330	22	+	0.043	d5	0.48	0	GGGGACGGAGGGAAGAAGAAAAA	-
ENSLCOT00000000027.1	486	505	19	+	0		0.37	0	AAAAGAAAAAGGGGAGAAG	-
ENSLCOT00000000027.1	22	47	24	-	0.04	d23	0.64	0	GCGGGAGGAGGGAAGGAAGGAGGGA	-
ENSLCOT00000000033.1	1026	1046	19	-	0.05	d14	0.4	0	GGAGGTGAAAAGGAAAAAAG	-
ENSLCOT00000000080.1	1639	1664	24	+	0.04	d16	0.52	0	GAGGGAAGAAGGAGAGCAAGGAGGA	-
ENSLCOT00000000093.1	3258	3280	21	+	0.045	d11	0.68	0	GGGGAGGGGGGTGAGAGGGAAA	-
ENSLCOT00000000093.1	1302	1323	20	-	0.048	d13	0.62	1	GGGGAAGTAGAGGAGGAGGGA	-
ENSLCOT00000000103.1	1311	1332	20	-	0.048	d13	0.62	1	GGGGAAGTAGAGGAGGAGGGA	-
